assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.test.js

```diff
@@ -433,4 +444,129 @@ describe( 'AudienceTilesWidget', () => {
 		).toBe( 1 );
 		expect( container ).toMatchSnapshot();
 	} );
+
+	it( 'should render correctly when there is partial data for Site Kit audiences', async () => {
+		registry.dispatch( MODULES_ANALYTICS_4 ).receiveGetSettings( {
+			availableAudiencesLastSyncedAt: ( Date.now() - 1000 ) / 1000,
+		} );
+
+		const configuredAudiences = [
+			'properties/12345/audiences/3', // New visitors
+			'properties/12345/audiences/4', // Returning visitors
+		];
+
+		const dates = registry.select( CORE_USER ).getDateRangeDates( {
+			offsetDays: DATE_RANGE_OFFSET,
+			compare: true,
+		} );
+
+		const dataAvailabilityDate = Number(
+			getPreviousDate( dates.startDate, -1 ).replace( /-/g, '' )
+		);
+
+		registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.receiveResourceDataAvailabilityDates( {
+				audience: {
+					'properties/12345/audiences/3': dataAvailabilityDate,
+					'properties/12345/audiences/4': dataAvailabilityDate,
+				},
+				customDimension: {},
+				property: {},
+			} );
+
+		registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.setAvailableAudiences( availableAudiences );
+
+		registry.dispatch( CORE_USER ).receiveGetAudienceSettings( {
+			configuredAudiences,
+			isAudienceSegmentationWidgetHidden: false,
+		} );
+
+		const newVsReturningReportOptions = {
+			...dates,
+			dimensions: [ { name: 'newVsReturning' } ],
+			dimensionFilters: {
+				newVsReturning: [ 'new', 'returning' ],
+			},
+			metrics: [
+				{ name: 'totalUsers' },
+				{ name: 'sessionsPerUser' },
+				{ name: 'screenPageViewsPerSession' },
+				{ name: 'screenPageViews' },
+			],
+		};
+
+		const newVsReturningReport = getAnalytics4MockResponse(
+			newVsReturningReportOptions
+		);
+
+		registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.receiveGetReport( newVsReturningReport, {
+				options: newVsReturningReportOptions,
+			} );
+
+		registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.finishResolution( 'getReport', [ newVsReturningReportOptions ] );
+
+		provideAudienceTilesMockReport( registry, configuredAudiences );
+
```

Comment:
We shouldn't need to provide this report once we've updated `useAudienceTilesReports()`.

```suggestion
```

Reply:
We will need other reports for the different metrics. So, I've modified the function.